app/views/article/index.html.erb

```diff
@@ -5,4 +5,6 @@
 		<th>TITLE</th>
 		<th>TEXT</th>
 	</tr>
+
+	<% @articles.all%>
```

Comment:
@GangadharSwami  
this will not work, change it to 

`Article.all`

Reply:
change it.